gitops/charts/postgres/postgres-test-application-values.yaml

```diff
@@ -319,7 +319,7 @@ pgBackRestConfig:
         - "ReadWriteOnce"
         resources:
           requests:
-            storage: 2Gi
+            storage: 5Gi
```

Comment:
This adjustment increases the backup PVC size which we were getting alerts on the PVC filling up.